### PR TITLE
Adding my Project

### PR DESCRIPTION
**Pull request by mezotv**
Added my open-sourced bot here for anyone who wants to contribute to a small bot! :D

### CODE REVIEW

**Review by ykdojo — CHANGES_REQUESTED** [review]
(explained my change request on stream)

**Comment by epicadidash** [comment]
@mezotv , I just want to say that your bot is okay . I went through it but its code can be reformatted and Please upgrade to discordjs's latest version as it will reduce many of your dependencies .

**Review by ykdojo — APPROVED** [review]

**Comment by epicadidash** [comment]
And also you can also add music bot features as well .

File: README.md [hunk 1]
```diff
@@ -238,5 +238,14 @@ https://github.com/do-community/cloud_haiku
 
 ---
 
+## Would You Bot
+> Would You is a Discord Bot built to help servers of any size gain more activity. Its code is fully open source and translated into German and English. The bot is asking users would you rather questions to get conversations started more easily.
```
Comment (ykdojo) [inline README.md:242]:
```suggestion
> Would You is a Discord Bot built to help servers of any size gain more activity. Its code is fully open source and translated into German and English. The bot is asks users *would you rather* questions to get conversations started more easily.
```

**Review by ykdojo — APPROVED** [review]

**Comment by mezotv** [comment]
@epicadidash We probably wont add music since thats not what the bot is made for. We will work on reformatting soon after the other stuff is done but if you want to you can contribute to it! Also gonna update discord.js soon!

**Pull request merged**